plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/common/ColorField.tsx

```diff
@@ -19,6 +19,11 @@ const ColorField: React.FC<Props> = ({ title, titleWidth, color, onChange }) =>
   const [text, setText] = useState(color);
   const [selectedColor, setSelectedColor] = useState(color);
 
+  useEffect(() => {
+    setText(color);
+    setSelectedColor(color);
```

Comment:
Is there any reason you need both text and selectedColor? It looks like this component uses both always together, right?

Reply:
Text is for whatever is written in the text box while selected color is only set when a valid color is entered.
![Screenshot 0005-02-20 at 10 32 24 AM](https://user-images.githubusercontent.com/35835415/219989717-749719fd-c985-4ab5-b681-cfe70230cf3c.png)